setup.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+# -*- encoding: utf-8 -*-
+"""Setup dot py."""
+from __future__ import absolute_import, print_function
+
+# import re
+from glob import glob
+from os.path import basename, dirname, join, splitext
+
+from setuptools import find_packages, setup
+
+
+def read(*names, **kwargs):
+    """Read description files."""
+    path = join(dirname(__file__), *names)
+    with open(path, encoding=kwargs.get('encoding', 'utf8')) as fh:
+        return fh.read()
+
+
+long_description = '{}\n{}'.format(
+    read('README.rst'),
+    read(join('docs', 'CHANGELOG.rst')),
+    )
+
+setup(
+    name='mdacli',
+    version='0.0.0',
+    description='A command line client for MDAnalysis Analysis classes.',
+    long_description=long_description,
+    long_description_content_type='text/x-rst',
+    license='GPLv2',
+    author='PicoCentauri, joaomcteixeira',
+    author_email='philip-loche@gmx.de, joaomcteixeira@gmail.com',
+    maintainer='PicoCentauri, joaomcteixeira, MDA devs',
+    maintainer_email='philip-loche@gmx.de, joaomcteixeira@gmail.com, mdanalysis@numfocus.org',
+    url='https://www.mdanalysis.org/',
+    packages=find_packages('src'),
+    package_dir={'': 'src'},
+    py_modules=[splitext(basename(i))[0] for i in glob("src/*.py")],
+    include_package_data=True,
+    zip_safe=False,
+    classifiers=[
+        'Development Status :: 3 - Alpha',
+        'Environment :: Console',
+        'License :: OSI Approved :: GNU General Public License v2 (GPLv2)',
+        'Intended Audience :: Science/Research',
+        'Natural Language :: English',
+        'Operating System :: POSIX',
+        'Operating System :: MacOS',
+        'Operating System :: Microsoft',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
+        'Topic :: Scientific/Engineering :: Bio-Informatics',
+        'Topic :: Scientific/Engineering :: Chemistry',
+        ],
+    project_urls={
+        'webpage': '',
+        'Documentation': '',
+        'Changelog': '',
+        'Issue Tracker': '',
+        'Discussion Forum': '',
+        },
+    keywords=[
+        # eg: 'keyword1', 'keyword2', 'keyword3',
+        ],
+    python_requires='>=3.7, <3.10',
+    install_requires=[
```

Comment:
Also here, I am not adding MDAv2 until it gets released. Otherwise tests will fail. This PR does not connect with the `results class` PR.